packages/botonic-plugin-hubtype-babel/src/hubtype-babel-api-service.ts

```diff
@@ -6,15 +6,15 @@ export class HubtypeBabelApiService {
     readonly host: string = 'https://api.hubtype.com'
   ) {}
 
-  async inference(text: string, accessToken: string) {
+  async inference(text: string, accessToken: string, hasSenseEnabled: boolean) {
```

Comment:
I don't think the `Enabled` is a good naming but better `includeHasSense` (see https://github.com/metis-ai/hubtype-backend/pull/1270/files#r1038188933).
also add a return type whith is an object with `intents` and, **optionally** `has_sense` boolean